src/Layout/Layout.js

```diff
@@ -240,7 +240,7 @@ Layout.propTypes = {
    * Defines the `align-items` style property.
    * It's applied for all the screen sizes.
    */
-  xsAlign: PropTypes.oneOf([
+  align: PropTypes.oneOf([ // eslint-disable-line react/sort-prop-types
```

Comment:
@oliviertassinari There's already precedence of sorts for using `eslint-disable-line react/sort-prop-types` with `onEnter`, `onEntering`, `onEntered` etc. being arranged in event order.